application/models/services/embeds/StandardEmbed.php

```diff
@@ -13,8 +13,8 @@ class StandardEmbed extends BaseEmbed
      */
     protected function getWrapper(string $placeholder = "PLACEHOLDER")
     {
-        $width = ($this->width + 20) . "px";
-        $height = ($this->height + 20) . "px";
-        return "<div style='width:{$width};height:{$height};border:1px solid red;'>haythem{$placeholder}</div>";
+        $width = ($this->embedOptions['width'] ?? $this->defaultWidth);
```

Comment:
We need to separate the width of the wrapper from the width of the structure around which it is wrapped. Similarly for height. So if your inner container has an immutable width and height, don't call it default and let's the wrapper have its own width and height. This way we separate concerns.